docs/docs/external/getting-started/interop/asset-exchange.md

```diff
@@ -55,15 +57,134 @@ run the following steps:
+  ```
+- Run the following to trigger `PartyB` locking `50` units of token type `t1` for `PartyA` in `Corda_Network` for 30 mins:
+  ```bash
+  CORDA_PORT=10009 ./clients/build/install/clients/bin/clients lock-asset --fungible --hashBase64=ivHErp1x4bJDKuRo6L5bApO/DdoyD/dG0mAZrzLZEIs= --timeout=1800 --recipient="O=PartyA,L=London,C=GB" --param=t1:50
+  ```
+  Note the `contract-id` displayed after successful execution of the command, will be used in next steps.
+- Run the following to verify `PartyB`'s lock (can be verified by both parties):
+  ```bash
+  CORDA_PORT=10006 ./clients/build/install/clients/bin/clients is-asset-locked --contract-id=<contract-id>
+  ```
+- Run the following to trigger `PartyA`'s claim for `50` units of token type `t1` locked by `PartyB` in `Corda_Network`:
+  ```bash
+  CORDA_PORT=10006 ./clients/build/install/clients/bin/clients claim-asset --secret=secrettext --contract-id=<contract-id>
+  ```
+  (_Note: Here the `PartyB` can see the node's log to get the revealed the hash preimage, to use it to claim bond in fabric network._)
+- Run the following to trigger `bob`'s claim for `bond01:a03` locked by `alice` in `network1`:
+  ```bash
+  ./bin/fabric-cli asset exchange-step --step=6 --recipient=bob --locker=alice --target-network=network1 --param=bond01:a03 --secret=secrettext
+  ```
+- Run the following to verify the status of the bond assets owned by `alice` and `bob` in the fabric network `network1` from `samples/fabric/fabric-cli` folder:
+   ```bash
+   ./scripts/getAssetStatus.sh
+   ```
+- Run the following to verify the status of the assets owned by `PartyA` and `PartyB` in the `Corda_Network` from `samples/corda/corda-simple-application` folder:
+  ```bash
+  ./scripts/getAssetStatus.sh
+  ```
+
+
+Above steps complete a successful asset exchange between Fabric and Corda networks. 
+In addition to above commands, following are the extra options:
+- _In case_, `alice` wants to unlock the bond asset, run the following to trigger `alice`'s re-claim for `bond01:a03` locked in `network1`:
+  ```bash
+  ./bin/fabric-cli asset exchange-step --step=7 --locker=alice --recipient=bob --target-network=network1 --param=bond01:a03
+  ```
+- _In case_, `PartyB` wants to unlock the token asset, run the following to trigger unlock for `t1:50` locked in `Corda_Network`:
+  ```bash
+  CORDA_PORT=10009 ./clients/build/install/clients/bin/clients unlock-asset --contract-id=<contract-id>
+  ```
+  
+## Corda with Corda
+
+We will demonstrate asset exchange of a tokens in `Corda_Network` with tokens on `Corda_Network2`. Here `PartyA` (`CORDA_PORT=10006`) and `PartyB` (`CORDA_PORT=10009`) in `Corda_Network` correspond to `PartyA` (`CORDA_PORT=30006`) and `PartyB` (`CORDA_PORT=30009`) in `Corda_Network2` respectively. Following are the step-by-step asset exchange process:
+
+  (_Note: the hash pair used in following steps can be replaced by any valid `sha256` hash pair_)
```

Comment:
Again, what's "hash pair"?

Reply:
hash, and its pre-image.

Reply:
Just use "hash" here. The preimage can be anything; only the hash can be generated using `sha265` or some other algorithm.